src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableV1.java

```diff
@@ -989,7 +996,9 @@ public ResultSet getResultSet(Properties info) throws SQLException {
    */
   private ResultSet getResultSetInternal(Properties info) throws SQLException {
     // Setup proxy info if necessary
-    SnowflakeUtil.setupProxyPropertiesIfNecessary(info);
+    this.httpClientKey = SnowflakeUtil.convertProxyPropertiesToHttpClientKey(ocspMode, info);
+
+    // TODO: add key to map
```

Comment:
nit: TODO

Reply:
Oops, thanks! Will remove this comment prior to merging.

Reply:
Removed!